mayalookassigner/app.py

```diff
@@ -31,26 +32,45 @@ def create_items(looks, assets):
     # Collect the looks we want to apply (by name)
     subsets = {look["subset"] for look in looks}
 
-    # Generate a queue item for every asset/look combination
-    items = []
+    # Collect the asset item entries per asset
+    # and collect the namespaces we'd like to apply
+    # todo: can we get the combined namespace filter an easier way?
+    assets = dict()
+    asset_namespaces = defaultdict(set)
+    for item in asset_items:
+        asset = item["asset"]["name"]
```

Comment:
This reads a bit confusing: `asset_name` would be a better variable name

Reply:
Thanks, fixed with https://github.com/Colorbleed/maya-look-assigner/pull/8/commits/ae66cebb5190ac13fb16a1a44b4880be458a4b7b